packages/react-liveness/package.json

```diff
@@ -65,6 +65,7 @@
   },
   "devDependencies": {
     "@aws-amplify/eslint-config-amplify-ui": "0.0.0",
+    "@aws-sdk/types": "3.310.0",
```

Comment:
Do we really have to depend on the aws-sdk types?

Reply:
We only need it for the IdentityProvider type, I copied that over and removed the dep on @aws-sdk/types, let me know what you think!

Reply:
In a library, dependencies tend to turn into tech debt.

Reply:
IMO we should directly use the types from the `aws-sdk` here in case `IdentityProvider` is updated

Reply:
Sync'd offline but we will copy these types over for now and will revisit when the types are exported from amplify auth